### PR TITLE
added member profile for aditya chatterjee fix for issue #1998

### LINKED ISSUE #1998: [Community] Member Profile: Aditya Chatterjee
**Current Behavior**

@warunicorn19 has been a consistent contributor and community member. :rocket: 

**Desired Situation**
Let's recognize @warunicorn19 as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @warunicorn19
- Twitter:
- LinkedIn: aditya-chatterjee-737729159
- Link to profile picture: https://drive.google.com/file/d/1mLojptCXkzVP9HYDJl5En9DQfmq9NQpn/view?usp=drivesdk

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

The CONTRIBUTING.md file has a "Common Types of Site Updates" section with detailed examples to follow here.[CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Jordan-Rob**
Signed-off-by: jordanrob709 <jordanrob709@gmail.com>

**Description**

This PR fixes #1998 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [* ] Yes, I signed my commits.

### CODE REVIEW

**Comment by warunicorn19** [comment]
@Tanuj22 Is the error due to the "-community" in the badges?

**Comment by sayantan1413** [comment]
Yes I also think so because we do not have a community badge I think.

File: src/collections/members/aditya-chatterjee/index.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name:       Aditya Chatterjee
+position: Contributor
+image_path: ./aditya-chatterjee.jpg
+github: warunicorn19
+linkedin: aditya-chatterjee-737729159
+location: Kolkata, India
+bio: 3rd Year CS undergrad of KIIT, Bhubaneswar. Navigating his way through the rapids of the programming world. I joined layer5 in July 2021, 
+    it's been an immense learning elivating not just knowledge but a way of life too.
+badges: - community
```
Comment (Tanuj22) [inline src/collections/members/aditya-chatterjee/index.mdx:10]: This section can be removed

Comment (warunicorn19) [reply src/collections/members/aditya-chatterjee/index.mdx:10]: You mean the badges section, or the entire section?

Comment (warunicorn19) [reply src/collections/members/aditya-chatterjee/index.mdx:10]: The badges section right?

Comment (Tanuj22) [reply src/collections/members/aditya-chatterjee/index.mdx:10]: Yes 😅

Comment (Jordan-Rob) [reply src/collections/members/aditya-chatterjee/index.mdx:10]: hahaha I thought you meant the location and bio too

**Comment by l5io** [comment]
🚀 Preview for commit 8736f84c0304396284cc508d4815efd2c82d97a6 at: https://60f8e782b6e9533a89c1a556--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
LGTM! Thanks @Jordan-Rob

**Review by Tanuj22 — APPROVED** [review]
Thank you @Jordan-Rob. Congrats @warunicorn19

**Comment by warunicorn19** [comment]
Thanks 😊, @Tanuj22

**Pull request merged**